### DISCUSSION TITLE
Looking for clarification regarding sandbox settings specifics

### DISCUSSION THREAD

**Discussion by shapeshifter83** [TEHE-Studios/ExpandedHelicopterEvents#200] [Q&A]
How does "Start Day", "Scheduler Duration", and "Month Since The Apocalypse" interact, specifically? FEATURES.md doc says it's "factored in", but that's as specific of information as I can find.

If I'm taking that blurb at face value, if I set my Months Since The Apocalypse to 12 months, then if I want helicopter events to proceed as normal in my game, I need to set Start Day to 360 (12x30) and my Scheduler Duration to whatever higher than 360.

Also, assuming this accounting for Months Since The Apocalypse is real, will the scheduler function at all if Start Day is set technically prior to my actual game start, i.e. Start Day set less than 360 when Months is set to 12?

**Comment by Chuckleberry-Finn** [comment]
Scheduler duration is days from start day. You need to manually account for months since the apocalypse when setting the start day.
